dev.zed.Zed-Preview.yaml

```diff
@@ -0,0 +1,208 @@
+# dev.zed.Zed-Preview.yaml
+app-id: dev.zed.Zed-Preview
+runtime: org.freedesktop.Sdk
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: zed-wrapper
+separate-locales: false
+finish-args:
+  - --device=dri
+  - --filesystem=home
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+
+  # Enable access to Flatpak host commands (Needs linter exception: https://docs.flathub.org/docs/for-app-authors/linter#exceptions)
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
needs exception

Reply:
are we the one to request an extension before the package is merged?

Reply:
the exception needs to be merged before it is accepted. Not sure why you marked this as resolved.

Reply:
https://docs.flathub.org/docs/for-app-authors/linter#exceptions

Reply:
Oh, okay. make a PR here: https://github.com/flathub-infra/flatpak-builder-lint/pull/638

Reply:
the build needs to be triggered in a few hours once the exception is up.

Reply:
alright!